README.md

```diff
@@ -35,7 +36,7 @@ You can then query this pod as you would with any other Kubernetes pod:
 
 ```
 $ kubectl logs -f registry-6wy8o
-$ kubectl exec -it registry-6wy8o psql
+$ kubectl exec -it registry-6wy8o sh
```

Comment:
Changed this to be "interact" with the kubernetes pod rather than query. Seems that it was copy-pasta anyway so I'm not against removing it.